data-transformers/transformers/statsTransformer.py

```diff
@@ -46,17 +46,21 @@
 lines = decompressed.readlines()
 data = sc.parallelize(lines)
 
-ob = json.loads(lines[0].decode())
 activeCpus = 0
-for c in ob["cpu_stats"]["cpu_usage"]["percpu_usage"]:
-    if c != 0:
-        activeCpus += 1
+for l in lines:
+    acpus = 0
+    ob = json.loads(l.decode())
+    for c in ob["cpu_stats"]["cpu_usage"]["percpu_usage"]:
+        if c != 0:
+            acpus += 1
+        if acpus > activeCpus:
```

Comment:
You don't need to do it inside the `for` loop, only once after the `for` loop should be enough.